Classes/ViewHelpers/ShowStatusViewHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace EWW\Dpf\ViewHelpers;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+class ShowStatusViewHelper extends \TYPO3\CMS\Fluid\Core\ViewHelper\AbstractViewHelper
+{
+
+    /**
+     *
```

Comment:
Maybe the view helper needs a little bit more documentation. ;-)

Reply:
Now the view helper returns a user friendly state name to be shown in the document list or the search result list. 